mappings/net/minecraft/world/entity/BasicEntityLookup.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_5578 net/minecraft/world/entity/BasicEntityLookup
```

Comment:
Maybe `SimpleEntityLookup`?

Or would this be a sort of `EntityLookupImpl`

Reply:
This is just a lookup composed of one id lookup and one collision lookup.

Reply:
+1 for `SimpleEntityLookup`

Reply:
Done